### PR TITLE
cv update with semantic html

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: staff/juan-sensio/cv/index.html [hunk 1]
```diff
@@ -1,42 +1,68 @@
...
-        <p>Git nivel avanzado (pull request legend).</p>
-        <p>Beber chupitos con el codo.</p>
-    </div>
+        <section id="educacion">
+            <h2>Educación</h2>
+            <section>
+                <h3>Skylab coders (www.skylabcoders.com). </h3>
+                <p><mark>Full-Stack</mark> Web Development Bootcamp.</p>
+            </section>
+            <aside>2018</aside>
```
Comment (manuelbarzi) [inline staff/juan-sensio/cv/index.html]: why not ```<time>``` instead of ```<aside>``` and located inside its own ```<section>```

Comment (ghost) [reply staff/juan-sensio/cv/index.html]: @manuelbarzi change done.

File: staff/juan-sensio/cv/index.html [hunk 2]
```diff
@@ -4,39 +4,65 @@
...
+            <h2>Habilidades</h2>
+            <p><mark>Git</mark> nivel avanzado (pull request legend).</p>
+            <p><mark>Git</mark> nivel avanzado (pull request legend).</p>
+            <p>Beber chupitos con el codo.</p>
+        </section>
+    </main>
+
+    <footer>
+        <p>Last update: <time>2018-04-06</time>.</p>
+        <nav>
```
Comment (alexgbor) [inline staff/juan-sensio/cv/index.html]: Nav is more useful for multiple links. I'd remove it and leave the anchor alone.

Comment (ghost) [reply staff/juan-sensio/cv/index.html]: @alexgbor change implemented.

**Pull request merged**